webapp/src/reducers/post_type.js

```diff
@@ -0,0 +1,18 @@
+import ActionTypes from 'action_types';
+
+export const postTypeComponent = (state = {}, action) => {
+    switch (action.type) {
+    case ActionTypes.REGIST_POST_TYPE_COMPONENT_ID:
+        if (action.data) {
+            const nextState = {...state};
+            if (action.data.postTypeComponentId == null) {
```

Comment:
consider `if (!action.data.postTypeComponentId)` or `if (Boolean(action.data.postTypeComponentId))`

Reply:
This intend to pass through empty string.
Because if user unregister custom post type component, the postTypeComponentId that is registered in state should be removed, and empty string will be set to `nextState.id`.

So `if (!action.data.postTypeComponentId)` and `if (Boolean(action.data.postTypeComponentId))` would be not valid here.